### ISSUE TITLE
OnreadyFind

### ISSUE THREAD

**Issue by zaksnet** [31/GodotOnReady#37]
Would be nice if we could also do something like (I'm including the optional parameters for reference):
```CSharp
[OnReadyFind("SomeNodeName"), Recursive = true, Owned = true] HBoxContainer _myContainer;
```

Which will essentially generate:
```CSharp
HBoxContainer _myContainer;
_myContainer = (HBoxContainer)FindNode("SomeNodeName", true, true)
```

**Comment by 31** [comment]
Thanks for filing! A few people in the C# Godot discord channel have asked about this too, and I'm not opposed to it.

(Personally, assigning paths in the editor has worked fine for me so far, but I've seen reasonable arguments about Find making it easier to move nodes around in the scene later.)

I'll add this when I have a bit.

**Comment by 31** [comment]
I've published version 1.2.0 with https://github.com/31/GodotOnReady/pull/39 to add this, although with the parameters inverted to keep the defaults `false` for simplicity reasons:

```cs
[OnReadyFind("SomeNodeName")]
[OnReadyFind("SomeNodeName", NonRecursive = true, Unowned = true)]
```

Thanks for https://github.com/31/GodotOnReady/pull/38! I added a comment explaining the ways/reasons I did it somewhat differently.

Let me know how it ends up working for you (or if it doesn't). 😄
